src/Directory.Build.props

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Nerdbank.GitVersioning" Version="3.2.31" PrivateAssets="all" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="all" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.205" PrivateAssets="all" />
```

Comment:
Ya, it's OK. It doesn't impact what the customer sees as our own nuget dependencies, and StyleCop.Analyzers comes out with stable releases _very_ slowly. C# 8, which we use, requires a beta stylecop.analyzers in order to avoid false warnings.